tests/suites/tenant/queryEditor/consts.ts

```diff
@@ -0,0 +1,29 @@
+export const longRunningQuery = `
```

Comment:
Please, add comment, that it's a long query that could cause "Memory limit exceeded" error, and that you should not run it on production DBs